test.js

```diff
@@ -13,6 +13,19 @@ t.tearDown(() => {
   })
 })
 
+test('throw on database open error v2', t => {
+  t.plan(3)
+
+  const fastify = Fastify()
+  fastify
+    .register(level, { name: '/test' })
+    .ready(err => {
+      t.ok(err)
+      t.equal(err.message, 'IO error: /test/LOCK: No such file or directory')
```

Comment:
I don't think this test would pass on Windows.
It is also trying to write to a file at the root of the file system, which is a bad practice.
I think you can get a similar error if you create a temp file (not directory) and try to open it.

Reply:
The idea was to write somewhere impossible (no permission), but I could trigger an error by setting `createIfMissing = false` in the [leveldown options](https://github.com/Level/leveldown/#options) and use "test" like in the other places.

Would that be ok?

Reply:
It would be ok as well, yes.